### ISSUE TITLE
S-101 document referencing older versions of ISO 191xx standards

### ISSUE THREAD

**Issue by forodd** [iho-ohi/S-101-Documentation-and-FC#131]
Don't know if this has been brought up before.

One of our "ISO-experts" had a look at the latest version of the S-101 PS (Edition 1.2.0.202301109) and noticed that the list of references in part 1.2 contains references to several ISO/TC211 standards that have been replaced by newer versions.

I know for sure that for some standards this is intentionally done so, but not sure its the case for all.

Some examples he found
ISO 19105:2022, (S-101 references 2000)
ISO 19107:2019 (S-101 references 2003)
ISO 19131:2022 (S-101 references 2008)
ISO 19157-1:2023 (S-101 references 2013)

Not sure this is an issue at all, and maybe not for 2.0.0 anyway, but comments anyone?

**Comment by DavidGrant-NIWC** [comment]
Agree this is not a pressing concern, but my comments are:

![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/1b7d8d7e-95f7-45bc-b3cf-3da990885095)

This should reference S-100 5.1. I don't think the ISO references are necessary unless they are explicitly referenced somewhere in the PS. The reference to S-100 should be enough since S-100 specifies the use of ISO standards:

### S-100 normative references
![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/a0c7f6bc-7cbe-42d1-a338-632e858d9014)

### S-101 vs. S-100 references to ISO standards listed in previous message
|Standard|S-101 agrees with S-100|Reference in S-100|
|:---:|:---:|:---|
|ISO 19105|Why is this referenced in S-101?|Unreferenced in S-100 other than in "Terms and definitions".
|ISO 19107|Yes|![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/b0f2fbcb-c439-46af-85c9-dca9a3bbd11c)
|ISO 19131|No|![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/4230fc45-f346-481e-a63b-64fa43ed0411)
|ISO 19157|No|![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/a0a3be02-5b98-4e3c-86f2-981aab83f7f4)|

**Comment by TomRichardson6** [comment]
I would agree with Dave that we should remove references unless we specifically reference something as S-101 refers to these standards via S-100. A short statement explaining this may be useful. I think one further action here could be to prompt the S-100WG to consider tracking where S-100 is behind the ISO standard and monitoring this, it may be that we choose to stay on the older versions but it should be a considered view.

**Comment by DavidGrant-NIWC** [comment]
> it may be that we choose to stay on the older versions but it should be a considered view.

In general, the newer standards add a tremendous amount of additional complexity with little benefit to S-100. Their use would impose additional implementation requirements on OEM's.

**Comment by JeffWootton** [comment]
Am preparing a Paper for S-100WG10 as I have done some checking and there are many inconsistencies in document references throughout the various parts of S-100 itself. Once this is resolved will look into removing the ISO Standards references from S-101 and instead referencing the appropriate section(s) of S-100.

**Comment by rmalyankar** [comment]
As we know, wholesale overhauls of S-100 in its entirety have been rare; I can't remember the last one.

Revisions in ISO TC211 standards sometimes change things in ways which have cascade effects on S-100 and consequently S-1xx and implementations, but don't add much value.

I do think the convention should to be use dated references to ISO standards, which at least formally stabilizes the foundations.

I'm not sure removing the ISO references altogether from S-1xx specifications would be a uniformly good thing; there are probably situations where indicating the underlying ISO standard does add value for the readers of the S-1xx document.

**Comment by JeffWootton** [comment]
**S-101PT16:** It was agreed that this was an issue with S-100 in general, and a paper should be prepared for submission to the S-100WG.

This was raised at the S-100WG10 meeting as part of the S-101PT Report. A further Paper is in preparation for the S-100 TSM12 meeting (March 2026).

Close Issue.
